careers.md

```diff
@@ -1,52 +1,32 @@
 ---
 layout: technical
-title: Carreers at Phoxygen
+title: Careers at Phoxygen
 ---
 
-# Phoxygen is hiring
+# Phoxygen is hiring!
 
-## Senior Javascript Developer at Phoxygen
+## Senior JavaScript Developer
 
-The technical team at Phoxygen is looking for experienced Javascript developer to work on Mozilla technologies. You'll be fully part of the team that work on Firefox OS
+Phoxygen is looking for an experienced JavaScript developer to work on Mozilla technologies. You’ll be fully part of the team that works on Firefox OS.
 
-### A few informations regarding your mission:
+### Mission
 
-- Translate UX design and wireframe in high quality code
-- Write and maintain performant, secured and reliable code
-- Make sure to understand and write you code with a sole objective, performance and reactivity of the frontend side
-- Know how to get along with the team and work in an efficient way remotely
-- Identify and fix bugs
-- Work in a peer-review fashion
-- Leark on workflow like github and bugzilla to fit in the company
+- Develop progressive web applications with optimal efficiency and performance
+- Translate UX designs and wireframes into high-quality code
+- Work remotely with a distributed team of passionate developers
 
-## Your qualification
+### Qualifications
 
-- Engineer diploma or equivalent professional experience.
-- More than 3 year as a software developer
-- Skilfully written and spoken english, mastering french is a plus, but not mandatory. Other mastered languages welcome.
+- Your open-source repositories are your best recommendation!
+- Engineering degree or equivalent professional experience
+- Successful 3+ year experience as a software developer
```

Comment:
I have the same doubt for this line, but I assumed the “no plural in compound names” rule applied here (as in “a three year old child”). If you can find a good reference for this rule, I’ll learn it once for all.